mindsdb/integrations/handlers/together_ai_handler/setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
```

Comment:
This is failing the automated dep check. I think we don't ask for these `setup.py` files anymore (@ZoranPandovski can you confirm)? So, please remove it and that way the check will pass, too.